packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -552,51 +544,74 @@ export interface WorkspaceFolderPickOptionsMain {
     ignoreFocusOut?: boolean;
 }
 
-export interface QuickInputTitleButtonHandle extends QuickTitleButton {
-    index: number; // index of where they are in buttons array if QuickInputButton or -1 if QuickInputButtons.Back
+// export interface QuickInputButtonHandle extends QuickInputButton {
+//     index: number; // index of where they are in buttons array if QuickInputButton or -1 if QuickInputButtons.Back
+// }
+
+export interface TransferQuickPickItems extends monaco.quickInput.IQuickPickItem {
```

Comment:
I Changed to `extends theia.QuickPickItem` although I implemented as in VSCode and `monaco.*` is used throughout plugin-ext - so why not refer to same entity as in VSCode? 
vscode:
![image](https://user-images.githubusercontent.com/71069170/117951316-ba3dca80-b31c-11eb-9d1f-e96b6235a4e5.png)

Theia:
![image](https://user-images.githubusercontent.com/71069170/117951616-00932980-b31d-11eb-9c5f-b15268e24a6e.png)

Reply:
For most types, we use DTO interfaces, but for some stuff, we use the monaco stuff in order to prevent the signature conversions. As far as using stuff from the theia API: using interfaces is fine, but classes won't work.

Reply:
I'm not sure I understand what your conclusion here is? you say `TransferQuickPickItems` can extend from `monaco.quickInput.IQuickPickItem` - like it is done in VS Code itself?